documentation/course/e2/README.md

```diff
@@ -2159,7 +2164,53 @@ TODO @piterek130: Dodać diagram pakietów, opis architektury i endpointy.
 
 ## Logistyka
 
-TODO @mlodybercik: Dodać diagram pakietów, opis architektury i endpointy.
+Do implementacji systemu obsługującej logistykę JakPrzyjade wybrano język programowania **TypeScript** z użyciem frameworka **NestJS**, w środowisku **Node.js**.
+
+Wybór ten podyktowany był głównie brakiem skomplikowanych operacji wykonywanych przez ten wycinek systemu oraz powszechnością języka TypeScript (również w zespole). Rozszerzenie języka JavaScript o statyczne typowanie pozwala na uniknięcie wielu błędów poprzez przerzucenie ich dużej części na etap kodowania, co jest istotne w przypadku systemu, który ma być zarówno wydajny jak i bezpieczny.
```

Comment:
```suggestion
Wybór ten podyktowany był głównie brakiem skomplikowanych operacji wykonywanych przez ten wycinek systemu oraz powszechnością języka TypeScript (również w zespole). Rozszerzenie języka JavaScript o statyczne typowanie pozwala na uniknięcie wielu błędów poprzez przerzucenie ich dużej części na etap kompilacji, co jest istotne w przypadku systemu, który ma być zarówno wydajny jak i bezpieczny.
```

Maybe like this?

- dynamic typing errors -> runtime
- static typing errors -> compilation